x-pack/plugins/apm/server/routes/alerts/rule_types/utils/get_filtered_series_for_preview_chart.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export type BarSeriesDataMap = Record<
+  string,
+  Array<{ x: number; y: number | null }>
+>;
+
+type BarSeries = Array<{
+  name: string;
+  data: Array<{
+    x: number;
+    y: number | null;
+  }>;
+}>;
```

Comment:
Can these be written using the `Coordinate` type:

```
export type BarSeriesDataMap = Record<string, Coordinate[]>;
type BarSeries = Array<{ name?: string; data: Coordinate[] }>;
```

Reply:
updated